c/test/cpp/auto_check_sbp_signing_MsgEcdsaSignatureDep.cc

```diff
@@ -0,0 +1,1389 @@
+/*
+ * Copyright (C) 2015-2021 Swift Navigation Inc.
```

Comment:
Looks like we might be missing the new test files for C and Java that cover the new message ID.